src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -497,6 +511,34 @@ public void openPrivateMessageTab(String username) {
    * Called from JavaScript when user clicked a URL.
    */
   public void openUrl(String url) {
+    if (url.contains(replayBaseUrl)) {
+      //wrong querry
+      replayService.findById(Integer.parseInt(url.replace(replayBaseUrl, ""))).thenAccept(replay -> {
+        Platform.runLater(() -> {
+          replayDetailController = uiService.loadFxml("theme/vault/replay/replay_detail.fxml");
+
+          replayDetailController.setReplay(replay);
+
+          Node replayDetailRoot = replayDetailController.getRoot();
+          replayDetailRoot.setVisible(true);
+          replayDetailRoot.requestFocus();
+
+          Stage stage = new Stage(StageStyle.UNDECORATED);
+          WindowController windowController = uiService.loadFxml("theme/window.fxml");
+          windowController.configure(stage, (Region) ((AnchorPane) replayDetailRoot).getChildren().get(0), true);
+          replayDetailController.setOnClosure(stage::close);
+          stage.setWidth(((Region) replayDetailRoot).getWidth());
+          stage.setHeight(((Region) replayDetailRoot).getHeight());
+
+          Stage mainStage = StageHolder.getStage();
+          stage.setX(mainStage.getX());
+          stage.setY(mainStage.getY());
+
+          stage.show();
+        });
+          });
```

Comment:
If you extract the  `thenAccept` block into a method with a descriptive name, the code becomes more readable:

```
      replayService.findById(Integer.parseInt(url.replace(replayBaseUrl, ""))).thenAccept(this::displayReplayDetail);
```

And if you set these in your commit-dialog, the code always gets indented and arranged properly:

![image](https://user-images.githubusercontent.com/1864183/30422617-cd404490-9940-11e7-87fd-e1aa56246f50.png)